Mathlib/CategoryTheory/Monoidal/Cartesian/Over.lean

```diff
@@ -185,4 +191,130 @@ lemma braiding_hom_left {R S : Over X} :
 lemma braiding_inv_left {R S : Over X} :
     (β_ R S).inv.left = (pullbackSymmetry _ _).hom := rfl
 
+variable {A B R S Y Z : C} [OverClass R X] {f : S ⟶ X}
+
+instance : (Over.pullback f).Braided := .ofChosenFiniteProducts _
+
+@[simps]
+instance canonicallyOverPullback : CanonicallyOverClass (Limits.pullback (R ↘ X) f) S where
+  hom := pullback.snd (R ↘ X) f
+
+@[simps! -isSimp mul one]
+instance monObjAsOverPullback [MonObj (asOver R X)] :
+    MonObj (asOver (Limits.pullback (R ↘ X) f) S) :=
+  ((Over.pullback f).mapMon.obj <| .mk <| asOver R X).mon
```

Comment:
This should say
```
@[simps! -isSimp mul one]
abbrev monObjMkPullbackSnd [MonObj (Over.mk f)] :
    MonObj (Over.mk (pullback.snd f g)) :=
  ((Over.pullback f).mapMon.obj <| .mk <| asOver R X).mon
```
And then you reintroduce this for schemes if necessary.

`OverClass` should not appear outside of Schemes.

Reply:
I must say this is a very confusing situation, since one shouldn't use `OverClass` for schemes either given that they have their own specialisation of it in the form of `Scheme.Over`